topics/proteomics/tutorials/metaquantome-taxonomy/tutorial.md

```diff
@@ -0,0 +1,391 @@
+>    - *"Mode"*: `Taxonomic analysis`
+>        - *"Plot type"*: `Volcano Plot (volcano)`
+>            - *"Name of the fold change column in the stat dataframe"*: `log2fc_T7_over_T2`
+>            - *"Name of the Corrected p-value column in the stat dataframe"*: `corrected_p_T7_over_T2`
+> 
+{: .hands_on}
+
+![T7_taxa](../../images/T7_taxa.png "Bar Chart visualization of Top 5 genus present in T7 sample."){: width="85%"}
+
+
+
+> ### {% icon hands_on %} Hands-on: Bar chart visualization of Taxonomy in T4 sample.
+>
+> 1. {% tool [metaQuantome: visualize](toolshed.g2.bx.psu.edu/repos/galaxyp/metaquantome_viz/metaquantome_viz/2.0.0-0) %} with the following parameters:
+>    - {% icon param-file %} *"Tabular file from metaQuantome stats or metaQuantome filter"*: `metaQuantome:stat` (output of **metaQuantome: stat** {% icon tool %})
+>    - {% icon param-file %} *"Samples file"*: `metaQuantome:create samples_file` (output of **metaQuantome: create samples file** {% icon tool %})
+>    - *"Mode"*: `Taxonomic analysis`
+>        - *"Plot type"*: `Bar Chart (bar)`
+>            - *"Taxonomic rank to restrict to in the plot"*: `genus`
+>            - *"Mean intensity column name"*: `T4_mean`
+>
+> 
+{: .hands_on}
+
+![T4_taxa](../../images/T4_taxa.png "Bar Chart visualization of Top 5 genus present in T4 sample."){: width="85%"}
+
+## *metaQuantome: visualize* Volcano Plots
+
+> ### {% icon hands_on %} Hands-on: Volcano Plot visualization of the data T4 and T2.
+>
+> 1. {% tool [metaQuantome: visualize](toolshed.g2.bx.psu.edu/repos/galaxyp/metaquantome_viz/metaquantome_viz/2.0.0-0) %} with the following parameters:
+>    - {% icon param-file %} *"Tabular file from metaQuantome stats or metaQuantome filter"*: `metaQuantome:stat` (output of **metaQuantome: stat** {% icon tool %})
+>    - {% icon param-file %} *"Samples file"*: `metaQuantome:create samples_file` (output of **metaQuantome: create samples file** {% icon tool %})
+>    - *"Mode"*: `Taxonomic analysis`
+>        - *"Plot type"*: `Volcano Plot (volcano)`
+>            - *"Name of the fold change column in the stat dataframe"*: `log2fc_T4_over_T2`
+>            - *"Name of the Corrected p-value column in the stat dataframe"*: `corrected_p_T4_over_T2`
+>        - {% icon param-select %} *"Flip the fold change (i.e., multiply log fold change by -1)"*: `No`
+> 
+>
+>    > ### {% icon comment %} Comment
+>    >
+>    > Here, we show metaQuantome's Taxonomy differential abundance volcano plot. The user may select the significance threshold (0.05 by default), and terms with statistically significant fold changes are colored green and labeled. However, we can currently only compare two conditions at a time. Here we have compared T4 and T2 time points. The user can also perform comparison of T7 and T2.
+>    {: .comment}
+>
+{: .hands_on}
+![T2-T4-taxa-volcano](../../images/T2-T4-taxa-volcano.png "Volcano plot for differentially expressed taxa comparing sample T4 and T2."){: width="100%"}
+
```

Comment:
@subinamehta  I think it would really helpful to add a section here for the second volcano plot comparing T7 and T2. I like that this step is already a part of the workflow, and I think users would find it useful to include the hands-on step here with the plot of the output so that they can compare to what they get when they go through the tutorial.